vue-custom-bootstrap/src/components/TextInput.vue

```diff
@@ -7,7 +7,8 @@
     class="mb-4"
   >
     <label>
-      <span :class="[ subLabel ? 'font-weight-bold': '', 'col-form-label']">{{label}}</span>
+      <!-- <span :class="[ subLabel ? 'font-weight-bold': '', 'col-form-label']">{{label}}</span> -->
+      <span class="question-label">{{label}}</span>
```

Comment:
You could delete this commented out code. 
`font-weight-bold` works here too.

Reply:
ok